### PR TITLE
Yelp more info/joe

### PR DESCRIPTION
**Pull request by joesalcedonyc**

### CODE REVIEW

File: group/src/main/java/com/android/group/network/YelpSingleton.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public final class YelpSingleton {
+    private static final String BASE_URL = "https://api.yelp.com/";
+    private static final String AUTH_TOKEN = "tPcRo3VZ6AO0qIxKNkWI2Fok77jRImOIJrl0rX15Or9k39vuC4QCLtT2BdD4cWjUgUw3flvp42af_UirQ00EYe69v7_S_yz_WNHGJLO5rlPnWIbt8QPM82j6EhrXXHYx";
+
+    private static Retrofit instance;
+
+    public static Retrofit getInstance() {
```
Comment (Moonianne) [inline group/src/main/java/com/android/group/network/YelpSingleton.java]: this method should be declared synchronized

File: group/src/main/java/com/android/group/view/recyclerview/AddLocationViewHolder.java [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
 import com.android.group.viewmodel.NetworkViewModel;
+import com.android.interactionlistener.OnFragmentInteractionListener;
 import com.squareup.picasso.Picasso;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class AddLocationViewHolder extends RecyclerView.ViewHolder {
 
     private ImageView venueImageView;
```
Comment (Moonianne) [inline group/src/main/java/com/android/group/view/recyclerview/AddLocationViewHolder.java]: try to have these views local to the method that assigns things to them

**Review by Moonianne — CHANGES_REQUESTED** [review]
just these 2 things and I think you are good to merge

File: group/src/main/java/com/android/group/network/YelpSingleton.java (hunk 1 above)
Comment (joesalcedonyc) [reply group/src/main/java/com/android/group/network/YelpSingleton.java]: All set.

File: group/src/main/java/com/android/group/view/recyclerview/AddLocationViewHolder.java (hunk 2 above)
Comment (joesalcedonyc) [reply group/src/main/java/com/android/group/view/recyclerview/AddLocationViewHolder.java]: All set.

**Review by Moonianne — APPROVED** [review]

**Pull request merged**